hackair-v2-advanced.ino

```diff
@@ -181,46 +186,43 @@ void setup() {
     delay(10000);
   }
 
-  //read updated parameters
+  // read updated parameters
   strcpy(hackair_api_token, custom_hackair_api_token.getValue());
   strcpy(sensebox_id, custom_sensebox_id.getValue());
   strcpy(osem_token, custom_osem_token.getValue());
 
-  //save the custom parameters to FS
+  // save the custom parameters to FS
   if (shouldSaveConfig) {
     Serial.println("saving config");
-    DynamicJsonBuffer jsonBuffer;
-    JsonObject& json = jsonBuffer.createObject();
-    json["hackair_api_token"] = hackair_api_token;
-    json["sensebox_id"] = sensebox_id;
-    json["osem_token"] = osem_token;
- 
+    DynamicJsonDocument jsonDoc(1024);
```

Comment:
Not sure about the 1024 here, but this is how it's done in the "How To Upgrade To 6.x" documentation.